content/docs/user-guide/dvc-files.md

```diff
@@ -159,6 +161,9 @@ stages:
     # User metadata and comments are supported.
 ```
 
+💡 You can parameterize this file with the special expression `${}`. See
```

Comment:
should it go before the example?

Reply:
I don't think so. The idea in this section is to introduce all the features of dvc.yaml by example first, and explain them later. The simple `${model_file}` substitution example aims to be self-explanatory (which BTW was missing from the 2nd stage's `cmd` but I just fixed that).